packages/react-i18n/src/i18n.ts

```diff
@@ -357,6 +357,13 @@ export class I18n {
     return firstName;
   }
 
+  hasCustomNameFormatter() {
```

Comment:
Can you add some documentation in the [readme](https://github.com/Shopify/quilt/tree/master/packages/react-i18n#i18n) and a changelog update for this as well.

Reply:
@ismail-syed Do you think the name of this method is appropriate for the desired context? All of the current custom name formatters use the Eastern naming order; Wouldn't it be more clear to replace the `CustomNameFormatter` keyword in variables/methods to communicate that instead?

Reply:
Sorry, I don't think I follow. 

Are you suggesting to use something like the following over what's there now?

```
hasCustomNameFormatter(locale, language) {}
```

Reply:
@ismail-syed the latest commit has the changes I implied. Please have a look! If it doesn't seem appropriate, I will revert them.